ui/slider.js

```diff
@@ -126,21 +128,19 @@ return $.widget( "ui.slider", $.ui.mouse, {
 				this.range = $( "<div></div>" )
 					.appendTo( this.element );
 
-				classes = "ui-slider-range" +
-				// note: this isn't the most fittingly semantic framework class for this element,
-				// but worked best visually with a variety of themes
-				" ui-widget-header ui-corner-all";
+				this._addClass( this.range, "ui-slider-range", "ui-widget-header" );
```

Comment:
Im not sure its not a theme class and its also not a structure class. So i do not think i would move to classes. But keeping the comment makes sense.